students/rgpag/lesson02/FizzBuzz.py

```diff
@@ -0,0 +1,9 @@
+for i in range(1,101):
+	if i%3 ==0 and i%5==0: 
```

Comment:
Please check your code for Pep8 (Python style guide) violations. Following the style conventions is a best practice that makes your code more readable for other developers. You can paste your code into an online [Pep8 checker](http://pep8online.com) or configure your editor to flag Pep8 violations as you type.